monai/apps/auto3dseg/auto_runner.py

```diff
@@ -209,6 +211,7 @@ def __init__(
         not_use_cache: bool = False,
         templates_path_or_url: str | None = None,
         allow_skip: bool = True,
+        mlflow_tracking_uri: str | None = None,
```

Comment:
I think having a method `set_mlflow_params(mlflow_tracking_uri=None, **kwargs)` might be better as we may need to modify more parameters of mlflow?

Reply:
Can you provide a few examples about which params can be modified?

Reply:
Okay.